pkg/volume/plugins.go

```diff
@@ -416,7 +416,7 @@ func NewPersistentVolumeRecyclerPodTemplate() *api.Pod {
 					Name:    "pv-recycler",
 					Image:   "gcr.io/google_containers/busybox",
 					Command: []string{"/bin/sh"},
-					Args:    []string{"-c", "test -e /scrub && echo $(date) > /scrub/trash.txt && find /scrub -mindepth 1 -maxdepth 1 -delete && test -z \"$(ls -A /scrub)\" || exit 1"},
+					Args:    []string{"-c", "test -e /scrub && rm -rf /scrub/..?* /scrub/.[!.]* /scrub/* ; echo $(date) > /scrub/trash.txt && exit 0 "},
```

Comment:
In the original bash, writing the date to 'trash' happened first to ensure something the volume be written to, not empty, and then verified empty afterwards.  You moved it to after the 'rm' command?  I don't see the confirmation that the dir is empty (checking via 'ls')